app/src/main/java/org/oppia/android/app/resumelesson/ResumeLessonFragment.kt

```diff
@@ -0,0 +1,88 @@
+package org.oppia.android.app.resumelesson
+
+import android.content.Context
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import org.oppia.android.app.fragment.InjectableFragment
+import javax.inject.Inject
+
+/** Fragment that allows the user to resume a saved exploration. */
+class ResumeLessonFragment :
+  InjectableFragment() {
+
+  companion object {
+    private const val RESUME_LESSON_FRAGMENT_INTERNAL_PROFILE_ID_KEY =
+      "ResumeExplorationFragmentPresenter.resume_exploration_fragment_internal_profile_id"
+    private const val RESUME_LESSON_FRAGMENT_TOPIC_ID_KEY =
+      "ResumeExplorationFragmentPresenter.resume_exploration_fragment_topic_id"
+    private const val RESUME_LESSON_FRAGMENT_STORY_ID_KEY =
+      "ResumeExplorationFragmentPresenter.resume_exploration_fragment_story_id"
+    private const val RESUME_LESSON_FRAGMENT_EXPLORATION_ID_KEY =
+      "ResumeExplorationFragmentPresenter.resume_Lesson_fragment_exploration_id"
+    private const val RESUME_LESSON_FRAGMENT_BACKFLOW_SCREEN_KEY =
+      "ResumeLessonFragmentPresenter.resume_lesson_fragment_backflow_screen"
+
+    /** Creates new instance of [ResumeLessonFragment].
+     * @param internalProfileId is used by the ResumeLessonFragment to retrieve saved checkpoint
+     * @param explorationId is used by the ResumeLessonFragment to retrieve saved checkpoint
+     */
```

Comment:
Please fix formatting (newline needed after initial ``/**`` and one blank line needed between the summary fragment and at-clauses).